bouncer/views.py

```diff
@@ -127,8 +131,21 @@ def error(self):
                              "fix it.")}
 
 
+@view.view_config(route_name='healthcheck', renderer='string')
+def healthcheck(request):
+    index = request.registry.settings['elasticsearch_index']
+    try:
+        status = request.es.cluster.health(index=index)['status']
+    except exceptions.ElasticsearchException as exc:
+        raise FailedHealthcheck('elasticsearch exception') from exc
+    if status not in ('yellow', 'green'):
+        raise FailedHealthcheck('cluster status was {!r}'.format(status))
+    return 'imok\n'
+
+
 def includeme(config):
     config.add_route("index", "/")
+    config.add_route("healthcheck", "/ruok")
```

Comment:
Because it's fun! 😛 

(Seriously, why risk conflicting with an actually useful URL path that we might use for something user-facing in the future...?)